google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableInstanceAdminStubSettings.java

```diff
@@ -503,9 +503,7 @@ public String extractNextToken(ListAppProfilesResponse payload) {
           ImmutableSet.copyOf(
               Lists.<StatusCode.Code>newArrayList(
                   StatusCode.Code.DEADLINE_EXCEEDED, StatusCode.Code.UNAVAILABLE)));
-      definitions.put(
-          "non_idempotent",
-          ImmutableSet.copyOf(Lists.<StatusCode.Code>newArrayList(StatusCode.Code.UNAVAILABLE)));
+      definitions.put("non_idempotent", ImmutableSet.copyOf(Lists.<StatusCode.Code>newArrayList()));
```

Comment:
Lists.newArrayList is no longer needed. Just use new ArrayList<>(). Also easier to read if there's an extra local variable for this.

Reply:
This code is autogenerated by gapic-generator. So it would have to change in its templates